src/Api.js

```diff
@@ -1,88 +1,123 @@
 var Api = {
   __host: {
     staging: {
-      v2: 'https://srv.wegostaging.com/v2',
-      v1: 'https://srv.wego.com'
+      v2: "https://srv.wegostaging.com/v2",
+      v1: "https://srv.wegostaging.com"
     },
     production: {
-      v2: 'https://srv.wego.com/v2',
-      v1: 'https://srv.wego.com'
+      v2: "https://srv.wego.com/v2",
+      v1: "https://srv.wego.com"
     }
   },
 
+  hotelEndpoints: {
+    searchHotelsUrl: function() {
+      return Api.getHost("v2") + "/metasearch/hotels/searches";
+    },
+    fetchHotelsUrl: function(searchId) {
+      return (
+        Api.getHost("v2") +
+        "/metasearch/hotels/searches/" +
+        searchId +
+        "/results"
+      );
+    },
+    searchSingleHotelUrl: function(hotelId) {
+      var path = "/metasearch/hotels/" + hotelId + "/searches";
+      return Api.getHost("v2") + path;
+    },
+    hotelDetailsUrl: function(hotelId) {
+      return Api.getHost("v1") + "/hotels/hotels/" + hotelId;
+    }
+  },
+
+  getHost: function(version = "v2") {
+    return this.__host[this.getEnvironment()][version];
+  },
+
   setEnvironment: function(env) {
     this.env = env;
   },
 
   getEnvironment: function() {
-    return this.env || Wego.ENV || 'staging';
+    return this.env || Wego.ENV || "staging";
   },
 
   searchTrips: function(requestBody, query) {
-    var uri = this.__host[this.getEnvironment()].v2 + '/metasearch/flights/searches';
+    var uri =
+      this.__host[this.getEnvironment()].v2 + "/metasearch/flights/searches";
```

Comment:
can we change the `.v2` into bracket notation?

Reply:
Or we can use `getHost()`

Reply:
ohh changes not need as it has been updated on [branch 42](https://github.com/wego/wego-sdk-js/pull/42/files#diff-dd58b2d4133ed7d8479cd745a1390f92R233)